docs/scripts.js

```diff
@@ -1,12 +1,31 @@
-$(document).ready(function() {
-  var preEls = $('pre');
+function slideExampleCode(e) {
+  e.preventDefault();
 
-  $('.example-code-link').click(function(e) {
-    e.preventDefault();
-    $(this).parent().next().slideToggle();
-  });
+  let slide = e.target.parentElement.nextElementSibling;
```

Comment:
You shouldn't use ES6+ in docs since old browsers are supported too.

Reply:
I can edit it if you want but it seems supported event with old browsers (after IE 8).